.woke.yml

```diff
@@ -1,4 +1,4 @@
 ignore_files:
   # Ignore ingress charm library as it uses non compliant terminology:
   # whitelist.
```

Comment:
Can you remove that comment which is not relevant any more, and add one on why ignoring apt lib ?